Monero_Payments.php

```diff
@@ -88,24 +88,21 @@ public function get_transfers(){
     }
      
      public function view_key(){
-    $query_key = array('key_type' => 'view_key');
-    $query_key_method = $this->_run('query_key', $query_key);
-    $this->_print($query_key_method);
+        $query_key = array('key_type' => 'view_key');
+        $query_key_method = $this->_run('query_key', $query_key);
+        $this->_print($query_key_method);
      }
      
+        /* A payment id can be passed as a string
+           A random payment id will be generatd if one is not given */
      public function make_integrated_address($payment_id){
-         if(isset($payment_id)){
-             $integrate_address_parameters = array('payment_id' => '');
```

Comment:
You actually don't need it! If the user does not pass any parameters into $payment_id then it will just stay empty and the wallet will generate a new one. This actually caused problems because, if you were going to use this, the if statement should use the ! operator like this: if(!isset($payment_id)) otherwise it will generate a random payment id instead of using the one that the user gives.